### PR TITLE
DOC: add omerc to projections list

### PR DESCRIPTION
**Pull request by bjlittle**
This PR is a follow-up to #2096.

See this [comment](https://github.com/SciTools/cartopy/pull/2096#issuecomment-1464864927).

### CODE REVIEW

File: docs/source/reference/projections.rst [hunk 1]
```diff
@@ -60,7 +59,7 @@ AzimuthalEquidistant
     import matplotlib.pyplot as plt
     import cartopy.crs as ccrs
 
-    plt.figure(figsize=(3, 3))
+    plt.figure(figsize=(3.0101, 3))
```
Comment (greglucas) [inline docs/source/reference/projections.rst:62]: Where did this value come from? Seems like it was OK on the current docs: https://scitools.org.uk/cartopy/docs/latest/reference/projections.html#azimuthalequidistant

Same with the Orthographic update.

Comment (bjlittle) [reply docs/source/reference/projections.rst:62]: Struck me as kinda odd too.

I simply done a `make projection` and this is the automated change

Comment (bjlittle) [reply docs/source/reference/projections.rst:62]: Did you want me to revert that particular auto-change?

Comment (greglucas) [reply docs/source/reference/projections.rst:62]: Ahh, the width is taken from the `x_limits` of each projection, so my guess is this has to do with PROJ updating some of their limits with a version bump at some point...

**Review by greglucas — APPROVED** [review]

**Comment by greglucas** [comment]
@bjlittle looks like this one didn't run the pre-commit.ci on the PR since it was in beforehand. Can you also fix the end-of-file in your upcoming PR? Might be a little more work and require a trailing newline in the `make_projection.py` script to be more robust for the future too.

**Comment by bjlittle** [comment]
Awesome, thanks @greglucas

**Pull request merged**